src/core/icons/fileIcons.ts

```diff
@@ -2876,5 +2876,6 @@ export const fileIcons: FileIcons = {
       name: 'minecraft-fabric',
       fileNames: ['fabric.mod.json'],
     },
+    { name: 'umi', fileNames: ['.umirc.ts', '.umirc.js'] },
```

Comment:
I recommend adding a pattern of Ecmascript files

```suggestion
    {
      name: 'umi',
      patterns: {
        ".umirc": FileNamePattern.Ecmascript,
      },
    },
```